content/pages/sprint/2024/08-coscup.rst

```diff
@@ -0,0 +1,229 @@
+===========================================
+Scisprint 2024 August in COSCUP
+===========================================
+
+:date: 2024-07-24 20:00
+:url: sprint/2024/08-coscup
+:save_as: sprint/2024/08-coscup.html
+
+Date
+-----
+
+* Date: 3rd August, Saturday, 2024
+* Time: 11:30 -- 16:00
+
+|
+
+Agneda 
+-------
+* 11:20-11:30 Opening
+
+* 11:30-12:00 Project introduction
+
+* 12:00-13:00 Career roundtable | Project coding session
+
+* 13:10-13:40 Tech sharing: Packet Party: Getting Down and Dirty with PcapPlusPlus
+
+* 13:40-14:10 Tech sharing: A High-Efficiency Natural Selection Simulator with Pybind11
+
+* 14:40-14:50 Tech sharing: Explore Practical Scientific Applications of Open-Source Large Language Models: RAG Knowledge Management and LLM Servitization Methods
```

Comment:
```suggestion
* 13:10-13:40 Tech sharing: `Packet Party: Getting Down and Dirty with PcapPlusPlus  <#Packet-Party:-Getting-Down-and Dirty-with-PcapPlusPlus>`__

* 13:40-14:10 Tech sharing: `A High-Efficiency Natural Selection Simulator with Pybind11 <#A-High-Efficiency-Natural Selection-Simulator-with-Pybind11>`__

* 14:40-14:50 Tech sharing: `Explore Practical Scientific Applications of Open-Source Large Language Models: RAG Knowledge Management and LLM Servitization Methods <#Explore-Practical-Scientific-Applications-of-Open-Source-Large-Language-Models:-RAG-Knowledge-Management-and-LLM-Servitization-Methods>`__
```

Reply:
Thanks, @rockleona. I will add links for the talks and update `Upcoming Event` for COSCUP and August sprint.

Reply:
Hi @YenPeiChen07 and @rockleona ,
I had open a [PR ](https://github.com/sciwork/swportal/pull/437)to edit the upcoming event. Currently, only DUTC NASA training event on the home page. I just noticed that the August Scisprint is on 8/17, earlier than the DUTC NASA training event. How do we arrange the event information in the home page?

Reply:
hmmm, @kuomc would you like to rebase your PR if this commit merge first?

Reply:
> hmmm, @kuomc would you like to rebase your PR if this commit merge first?

I agree.